### ISSUE TITLE
Packet inspector breaks with Vanilla Server

### ISSUE THREAD

**Issue by ferrohd** [valence-rs/valence#102]
I've tried to use `packet_inspector` with `conway` and it works just fine.

But when using a vanilla server, the inspector errors out with:
```
Connection to 127.0.0.1:45158 ended with: decoding packet: failed to read field `verify_token` from struct `EncryptionRequest`: Length of array is out of bounds while decoding (got 4, needed 16..=16)
```

The server is PaperMC `1.19.2-183` and I'm using a vanilla client

**Comment by dyc3** [comment]
This is fixed with #65

**Comment by ferrohd** [comment]
Sorry, didn't see it. Closing.
